mlflow/server/auth/__init__.py

```diff
@@ -801,11 +801,12 @@ def delete_registered_model_permission():
     return make_response({})
 
 
-def _enable_auth(app: Flask):
+def enable_auth(app: Flask = app):
```

Comment:
The goal is to avoid accidental app mutation. In the original code, touching this module mutates the app. In the new code, it doesn't.

Reply:
i think this looks great and is much easier to debug